test/e2e/resources/layer_version.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: lambda.services.k8s.aws/v1alpha1
 kind: LayerVersion
 metadata:
-  name: $LAYER_VERSION
-  annotations:
-    services.k8s.aws/region: $AWS_REGION
+  name: v1
+  annotations: 
+    services.k8s.aws/region: us-west-2
 spec:
-  layerName: $LAYER_VERSION
+  layerName: v1
   content:
-    s3Bucket: $BUCKET_NAME
-    s3Key: $LAMBDA_FILE_NAME
+    s3Bucket: 322962841005-lambda-test
+    s3Key: my-deployment-package.zip
```

Comment:
I think the reason e2e tests are failing is that you forgot to undo the changes made to this file.

Reply:
Done. Thanks